Makefile

```diff
@@ -144,7 +144,7 @@ docker-func-test: docker     ## Run the built CNI container image to use in func
 # Run unit tests
 unit-test: export AWS_VPC_K8S_CNI_LOG_FILE=stdout
 unit-test:    ## Run unit tests
-	go test -v -coverprofile=coverage.txt -covermode=atomic $(ALLPKGS)
+	 go test -v -mod=mod -coverprofile=coverage.txt -covermode=atomic ./pkg/...
```

Comment:
should -mod=mod replaced with `VENDOR_OVERRIDE_FLAG`?

Reply:
Yeah, will update it.